Gruntfile.js

```diff
@@ -81,15 +109,22 @@ module.exports = function(grunt) {
         }
     });
 
+    grunt.registerTask('css', ['less:development', 'less:production']);
+
     grunt.registerTask('publish', 'Publish from CLI', [
         'concat',
         'uglify',
+        'less:production',
         'gh-pages:publish'
     ]);
 
     grunt.registerTask('deploy', 'Publish from travis', [
         'concat',
         'uglify',
+        'css',
         'check-deploy'
     ]);
+
+    grunt.registerTask('delta', ['deploy', 'watch'])
```

Comment:
hi @Lenninlasd I don't think we want to minify/uglify the js this would make debugging much harder. 
Wouldn't it be better to just add `css` to `delta`? And is there a better name for `delta` ?